### ISSUE TITLE
Automation task starter for Art

### ISSUE THREAD

**Issue by rifaterdemsahin** [rifaterdemsahin/aif#261]
He works on the RPA tools
![image](https://user-images.githubusercontent.com/5270036/44943661-0c836b00-adc2-11e8-8fc5-5de1fd45bfec.png)

**Comment by rifaterdemsahin** [comment]
![image](https://user-images.githubusercontent.com/5270036/44943682-5e2bf580-adc2-11e8-9e25-0b73de57356a.png)

**Comment by rifaterdemsahin** [comment]
Hotkey is the program.

**Comment by rifaterdemsahin** [comment]
https://github.com/rifaterdemsahin/aif/wiki/Freelancer-task-delegation

**Comment by Argimko** [comment]
ok, got it

**Comment by rifaterdemsahin** [comment]
![image](https://user-images.githubusercontent.com/5270036/44943786-2160fe00-adc4-11e8-8076-e755226fc7f0.png)

**Comment by rifaterdemsahin** [comment]
Here is the video of the task detailed in here
https://drive.google.com/file/d/1iub21bFrRYR-jcXb96pULqBB3yy7JBpg/view?usp=sharing

**Comment by rifaterdemsahin** [comment]
85 ?

**Comment by Argimko** [comment]
Yes, Mr. Sahin, my price is 6 times higher than some programmers. My clients are ready to pay this price because they appreciate highly qualified professionals, work with which saves time for clients.

And if you found highly qualified programmers with low price – it is a tribute to your persistence.
